common/app/navigation/helpers/helpers.scala

```diff
@@ -16,8 +16,9 @@ object NavigationHelpers {
     val editionId = edition.id.toLowerCase()
 
     NavLinkLists(List(
-      NavLink("become a supporter", getReaderRevenueUrl(Membership, SideMenu)),
-      NavLink("subscribe", getReaderRevenueUrl(Subscribe, SideMenu))
+      // This methods can be reverted once we decide to deploy the new support site to the rest of the world.
```

Comment:
Thanks for the comment! But I think this comment is misleading here. Maybe add it [here](https://github.com/guardian/frontend/pull/18002/files#diff-4f5ed730ef009a2754059075ea00df85R126) instead? I wouldn't want this list to be deleted.